css-view-transitions-1/Overview.bs

```diff
@@ -466,6 +486,13 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 			Initially a new [=style sheet=] in the [=user-agent origin=], ordered after the [=HTML user agent style sheet=].
 
 			Note: This is used to hold dynamic styles relating to transitions.
+
+		: <dfn>view-transition root pseudo-element</dfn>
```

Comment:
nit: Do we need this here? We should be able to refer to this by active DOM transition's transition root pseudo-element.

Reply:
That's a very good point. In fact, it's kinda silly that this isn't cleared as part of "Clear view transition". I'll fix that up.

Reply:
Oh, now I know why.

We don't add the "view-transition root pseudo-element" at the same time as the active transition. The document's "view-transition root pseudo-element" is used to define when this pseudo element is in the document.

Reply:
I'll change it to a boolean to avoid duplicated info